support-models/src/main/scala/com/gu/support/workers/PaymentMethods.scala

```diff
@@ -4,11 +4,13 @@ import cats.syntax.functor._
 import com.gu.i18n.Country
 import com.gu.support.encoding.Codec
 import com.gu.support.encoding.Codec.capitalizingCodec
+import com.gu.support.zuora.api.{DirectDebitGateway, PayPalGateway, PaymentGateway}
 import io.circe.syntax._
 import io.circe.{Decoder, Encoder}
 
 sealed trait PaymentMethod {
   def `type`: String
+  def paymentGateway: PaymentGateway
```

Comment:
now the actual Zuora payment gateway name is defined when the zuora payment method is determined.  Before it was determined at the point of writing the subscription to zuora based on this object type and the currency.